com.unity.netcode.adapter.utp/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this package will be documented in this file. The format
 
 ### Changed
 
+- Updated links to the UTP package docs, this is a backport of PR-1603 (#1604)
```

Comment:
not needed (also not in the original PR)

Reply:
Reverted